### DISCUSSION TITLE
Not deploying latest commits, despite commit showing up in the "project overview" UI

### DISCUSSION THREAD

**Discussion by AntonOsika** [vercel/community#2506] [Bug Report]
Reproduction steps:
- Push change to main

Expected behavior:
- Deploys changes to production

Actual behavior:
- No new deployment

Additional details:

It's very odd, but the latest commit it showing up in the project overview (see first screenshot, with commit 22 min ago) but not in the project itself (see second screenshot, with commit 50 min ago).

![image](https://github.com/vercel/community/assets/4467025/3536791a-34cd-482b-873a-ebecd6e510ad)
![image](https://github.com/vercel/community/assets/4467025/3cf95544-c06a-47a4-b133-54b0e09bca83)

**Comment by AntonOsika** [comment]
Seems to work now, even though I'm sure there was a bug here.

Closing for now, but might be worth investigating since I'm sure I wasn't alone:) @amyegan 


(Might have been that I changed name of project etc, and I hadn't given access to all github repos (weird anyways, since Vercel was clearly able to see the most recent commits judging from screenshot))

**Reply by amyegan** [reply]
Glad to hear it's working now. Hopefully it was just a short delay communicating between different services. Please let me know if you see this problem again 🙏
